Fix for cached files from arvados-controller

chlige opened the pull request:
Updated handling of cached files in arvados-controller.   Fixes issue of serving of discovery document.    See discussion at https://matrix.to/#/!iCGZYZrhxQUZbnMoxM:gitter.im/$EXPq511LkcGX3bZ2khs--y-hfNhKPbPZ2PzM3ZbpX40?via=gitter.im&via=matrix.org

tomclegg commented:
Hi, thanks for the contribution!

Could you update your commit message with a Developer Certificate of Origin line like this? See [CONTRIBUTING.md](https://github.com/arvados/arvados/blob/main/CONTRIBUTING.md)

```
Arvados-DCO-1.1-Signed-off-by: Joe Smith <joe.smith@example.com>
```

Our git hooks won't let us merge a commit without this.

I have slightly more extensive fix to add to this, but I'd rather give you credit by including your commit first.

chlige commented:
Tom -

Sorry about that.  I have updated the message for the Git commit.  I also added my name to the AUTHORS file according to (https://github.com/arvados/arvados/blob/main/CONTRIBUTING.md) .  However, if you would rather omit me from the AUTHORS, that would be perfectly fine since this is only a simple fix.

- George
